lib/codebreaker/view.rb

```diff
@@ -0,0 +1,96 @@
+module Codebreaker
+  class View
+    PLAYER_NAME = 'Player'.freeze
+    def initialize
+      @messages = Messages.new
+      @game = Game.new
+      @moves_count = Game::COUNT_MOVES
+      view_result_games
+    end
+
+    def load_game
+      player_name
+      @messages.print_info(@name_player)
+      @game.start_game
+      steps
+      @game.data_preparation(@name_player)
+      message_of_result
+      save_result
+      view_result_games
+      plays_again
+    end
+
+    private
+
+    def steps
+      loop do
+        break if @game.loses_game?
+        case @game.player_code = read_input
+        when 'q' then exit
```

Comment:
I don't like approach with `case`. What will be if quantity of commands will be more than 10?. In future if you will have time please refactor this. For example, instead of `case`, you can use approach with describing all your commands in hash and then call this commands (useful article where you can read about this http://bobnadler.com/articles/2010/08/02/refactoring-case-statements-in-ruby.html).